index.html

```diff
@@ -34,9 +34,28 @@ <h1 class="position-text">Don't let your business be left behind. <br>
             Scale up with VED.
             </h1>
         </section>
-        <section id="about">
-            <h2>About</h2>
 
+        <section id="about">
+            <h1 class="about-title center">About Us</h1>
```

Comment:
We're already using h1 in the hero section of the page, so we'd have to make this a h2.

Reply:
Ok i'll resolve this with a h2 😄